metricflow/object_utils.py

```diff
@@ -1,15 +1,14 @@
 from __future__ import annotations
-import itertools
 import logging
 import pprint
 import textwrap
 from collections.abc import Mapping
 from dataclasses import is_dataclass, fields
-import datetime
 from hashlib import sha1
-from typing import Sequence, TypeVar, Tuple, Union
+from typing import Sequence
 
 from metricflow.model.objects.base import HashableBaseModel
+from metricflow.sql.sql_column_type import SqlColumnType
```

Comment:
I don't think we want this here, right? But of course then I don't know where hash_items should live. Hmmm.

That will get easier after #444 is merged, so maybe we just leave this here for now.